### PR TITLE
Rework kevlar, add slash resistant and EOD gear

### PR DESCRIPTION
**Pull request by ymber**
#### Summary
SUMMARY: Content "Rework aramid materials, add slash resistant and EOD gear"

#### Purpose of change
Our current kevlar materials don't behave like kevlar.

#### Describe the solution
Adds new materials for different types of aramid fabric and armor panels and adds some new slash resistant gear and EOD gear using them.

#### Testing
Game loads. New items don't crash my debug menu. Values for stuff look about right.

#### Additional context
This is just #35522 rebased onto current master.

### CODE REVIEW

File: data/json/items/armor/gloves.json [hunk 1]
```diff
@@ -550,7 +550,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": { "str": "pair of tactical gloves", "str_pl": "pairs of tactical gloves" },
-    "description": "A pair of reinforced Kevlar tactical gloves.  Commonly used by police and military units.",
+    "description": "A pair of flame and cut resistant aramid fabric gloves.  Commonly used by police and military units.",
```
Comment (LaVeyanFiend) [inline data/json/items/armor/gloves.json:553]: These need an "environmental_protection" value to have fire resistance

File: data/json/materials.json [hunk 2]
```diff
@@ -770,6 +770,28 @@
...
     "name": "Kevlar",
+    "density": 9,
+    "soft": true,
+    "reinforces": true,
+    "specific_heat_liquid": 0.82,
+    "specific_heat_solid": 0.45,
+    "latent_heat": 273,
+    "bash_resist": 1,
+    "cut_resist": 3,
+    "acid_resist": 5,
```
Comment (kevingranade) [inline data/json/materials.json:781]:
```suggestion
    "bullet_resist": 5",
    "acid_resist": 5,
```

File: data/json/items/tool_armor.json [hunk 3]
```diff
@@ -786,6 +786,51 @@
...
+    "id": "armor_EOD",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "copy-from": "armor_EOD_light",
+    "name": "EOD suit",
+    "description": "This is a massive suit constructed from kevlar and nomex for explosive ordnance disposal, complete with an armored full face helmet and boots.  This garment will provide significant protection from overpressure, fragmentation, impact and heat.  The electronically shielded helmet contains a camera, a two-way radio, and a headlamp, all of which can be voice-activated for redundancy.  This likely has nothing to do with the suit's lack of gloves.",
+    "weight": "31298 g",
+    "volume": "10000 ml",
+    "price": 760000,
+    "covers": [ "HEAD", "MOUTH", "EYES", "TORSO", "ARMS", "LEGS", "FEET" ],
```
Comment (Amoebka) [inline data/json/items/tool_armor.json:799]: Which suit is this based on? Every real bomb suit I know of has detachable boots and helmet. Please avoid making "covers everything" single pieces unless there's a strong reason to, it's both less realistic and screws over mutated characters.

Comment (ymber) [reply data/json/items/tool_armor.json]: Split up the items into most of their individual parts. They're based on the Med-Eng EOD 10 and TAC6E.

File: data/json/items/tool_armor.json [hunk 4]
```diff
@@ -786,6 +786,53 @@
...
+    "material_thickness": 15,
+    "flags": [ "STURDY", "OUTER", "RAINPROOF", "TWO_WAY_RADIO", "WATCH", "ALARMCLOCK" ]
+  },
+  {
+    "id": "helmet_eod_on",
+    "type": "TOOL_ARMOR",
+    "copy-from": "helmet_eod",
+    "name": "EOD helmet (on)",
+    "extend": { "flags": [ "LIGHT_350", "CHARGEDIM", "TRADER_AVOID" ] },
+    "power_draw": "10 W",
```
Comment (ZhilkinSerg) [inline data/json/items/tool_armor.json:832]:
```suggestion
    "power_draw": 10000,
```

File: data/json/items/armor/arms_armor.json [hunk 5]
```diff
@@ -279,5 +279,16 @@
     "material_thickness": 3,
     "environmental_protection": 1,
     "flags": [ "STURDY", "BELTED", "WATER_FRIENDLY" ]
+  },
+  {
+    "id": "armguard_cut_resistant",
+    "type": "ARMOR",
+    "name": "pair of cut-resistant arm sleeves",
```
Comment (ZhilkinSerg) [inline data/json/items/armor/arms_armor.json:286]: All new names need to be be updated with `"name": { "str": "pair of cut-resistant arm sleeves", "str_pl": "pairs of cut-resistant arm sleeves" }` syntax.

File: data/json/items/armor/boots.json [hunk 6]
```diff
@@ -145,6 +145,48 @@
...
+    "color": "light_gray",
+    "environmental_protection": 2,
+    "material_thickness": 7,
+    "flags": [ "STURDY", "OUTER" ]
+  },
+  {
+    "id": "toecaps",
+    "type": "ARMOR",
+    "name": "pair of toecaps",
+    "name_plural": "pairs of toecaps",
```
Comment (ZhilkinSerg) [inline data/json/items/armor/boots.json:172]: There are still some occurrences of `name_plural` strings instead of `name` object with `str_pl` inside.

**Pull request merged**